test/functional/fixtures/api/es-next/selector/test.js

```diff
@@ -271,6 +275,51 @@ describe('[API] Selector', function () {
                     });
             }
         );
+
+        it('Should raise error if addCustomMethods method argument is not object',
```

Comment:
We used it initially, but we don't do that anymore. There is no need for duplication.